data-serving/scripts/import-data/src/index.ts

```diff
@@ -1 +1,71 @@
-new Mongo()
\ No newline at end of file
+declare let args: SetupDatabaseParameters;
+
+interface SetupDatabaseParameters {
+    databaseName: string;
+    collectionName: string;
+    schemaPath: string;
```

Comment:
nit, up to you but to simplify things a bit perhaps you can import the ./../../data-service/schemas/cases.schema.json directly?

https://mariusschulz.com/blog/importing-json-modules-in-typescript#importing-json-files-via-static-import-declarations

We can parameterize things later of if we need to, or not.

Reply:
we def can -- was trying to leave it parameterized in case we wanted to move this up a dir and reuse for other collections (sources, users)

if we're not parameterizing that, is there any point to parameterizing the db/collection names, either? seems like those won't change for cases

Reply:
okay then maybe let's see how much we can reuse once we run it for another kind of collection, for now let's keep it as is.

Reply:
btw for posterity, if we do keep the parameterization, this is the syntax for `npm run setup` (as opposed to `npm run setup-cases`

`DB="somedb" COLL="somecollection" SCHEMA="../some/schea/path.json" npm run setup`